### DISCUSSION TITLE
How might we add a Flipper plugin?

### DISCUSSION THREAD

**Discussion by djMax** [LegendApp/legend-state#117] [Ideas]
It's quite nice to see diagnostic info in a debugger like Flipper. For MST, this is basically any state transition. Most of the tracing here now is "locked" to output I think? Might be better to have some intermediate event processor that could be hooked up to Reactotron/Flipper/etc?

**Comment by jmeistrich** [comment]
Yes, that would be great! I don't know anything about how intermediate event processors work though. Do you know of a good resource to get up to speed on that?

The current tracing functions run on the listener side because they're meant to be used as React hooks, but in this case we'd want it to run on the notifying side. That would be very easy to add, just need to know what format it would need to be in.
